benchmarks/hash_anchored_vs_plain.py

```
@@ -0,0 +1,428 @@
+                        end = line.index("]", start) + 1
+                        landed_para = line[start:end]
+                    break
+
+            if landed_para == marker:
+                plain_correct += 1
+            elif landed_para:
+                plain_wrong += 1
+                print(f"    occ={old_occ}: targeted {marker}, landed in {landed_para} ← WRONG")
+            else:
+                plain_correct += 1  # Can't verify
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)